README.md

```diff
@@ -148,18 +149,22 @@ jupyter notebook
 --->
 
 # <a name="models"></a>
+# Supported Models
 
-# Models
+Any models that are trained on datasets in Python `numpy.array`, `pandas.DataFrame`, `iml.datatypes.DenseData`, or `scipy.sparse.csr_matrix` format are supported by this API.
+
+The explanation functions accept both models and pipelines as input. If a model is provided, the model must implement the prediction function predict or predict_proba that conforms to the Scikit convention.  If a pipeline script is provided, the explanation function assumes that the running pipeline script returns a prediction. The repository also supports models trained via PyTorch, TensorFlow, and Keras deep learning frameworks.
```

Comment:
Rephrase this as
The explanation functions accept both models and pipelines as input, as long as the model or pipeline implements the prediction function predict or predict_proba that conforms to the Scikit convention.

Reply:
On "The repository also supports models trained via PyTorch, TensorFlow, and Keras deep learning frameworks.", @imatiach-msft can you suggest edit on this? We support these because we provide wrapper on these model to make their API Scikit compliant?

Reply:
@vingu I believe this is true. As long as they can wrap it up for us to conform to predict and predict proba, we can support them. We did that for TFWorld.